packages/ingest/test/test-FtpProviderClient.js

```diff
@@ -132,7 +133,7 @@ test('useList is present and true when assigned', async (t) => {
 
   await myFtpProviderClient.list('');
 
-  t.true(jsftpSpy.callCount > 1);
+  t.true(jsftpSpy.callCount > 0);
```

Comment:
I tried grokking the code, but why did the expected call count drop?

Reply:
It actually went up. In my previous PR I changed this because the recursive calls caused the callCount to go from `1` to `13`, so I set `> 1`. Then I made a change here and the number is back to one, _I think_. It's been a minute since I changed that. It's a number that may or may not increase as the complexity of `test-data` increases, since we mount that as a volume on the ftp server, but in all cases should be > 0.